cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/servlet/SessionManager.java

```diff
@@ -51,8 +52,8 @@
   private final Time _time;
   private final ScheduledExecutorService _sessionCleaner =
       Executors.newSingleThreadScheduledExecutor(new KafkaCruiseControlThreadFactory("SessionCleaner",
-                                                                                     true,
```

Comment:
Looks like this change modifies and breaks the indentation formatting. Can we avoid such changes?
Importing the project style file on IntelliJ may help: https://github.com/linkedin/cruise-control/blob/master/docs/code-style.xml

Reply:
Will do. Thanks